urwid/display/_raw_display_base.py

```diff
@@ -554,14 +588,16 @@ def draw_screen(self, size: tuple[int, int], r: Canvas) -> None:
 
         logger.debug(f"Drawing screen with size {size!r}")
 
-        o = [escape.HIDE_CURSOR, self._attrspec_to_escape(AttrSpec("", ""))]
+        last_attributes = None  # Default = empty
+
+        output = [escape.HIDE_CURSOR, attr_to_escape(last_attributes)]
```

Comment:
`attr_to_escape` with `None` should set default attributes. It's enough for normal and incremental draw.